core/utils.py

```diff
@@ -418,6 +419,27 @@ def convert_png_data_url_to_binary(image_data_url: str) -> bytes:
         raise Exception('The given string does not represent a PNG data URL.')
 
 
+def convert_webp_data_url_to_binary(image_data_url: str) -> bytes:
```

Comment:
Hmm, this is almost the same as the PNG version above, can we parametrize this?

Reply:
And have just one function?

Reply:
Updated